### PR TITLE
Fix #4146 - add whats new doc

### LINKED ISSUE #4146: No what's new in v6.2 document
Issue Details
-----------------
Missing a what;s new in v6.2 document

Version(s) of document impacted
------------------------------
- [x] Impacts 6.next document
- [x] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Looks like we are missing a what's new in PowerShell v6.2 from the table in https://docs.microsoft.com/en-us/powershell/scripting/whats-new/what-s-new-in-powershell-core-60?view=powershell-6

@joeyaiello / @SteveL-MSFT - can we make sure that for 6.3/7 this is included as part of the pre-release tasks to complete as not to miss this in future?

### PR DESCRIPTION
**Pull request by sdwheeler**
@joeyaiello please review

### CODE REVIEW

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[63a541f](https://github.com/sdwheeler/PowerShell-Docs/pull/4170/commits/63a541fbbffb3e92109f7c26b03efb8f82d5869e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/toc.yml](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4146/reference/docs-conceptual/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-3.0&branch=pr-en-us-4170) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-4.0&branch=pr-en-us-4170) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-5.0&branch=pr-en-us-4170) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-5.1&branch=pr-en-us-4170) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-6&branch=pr-en-us-4170) (powershell-6) |
[reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-62.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4146/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-62.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-3.0&branch=pr-en-us-4170) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-4.0&branch=pr-en-us-4170) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-5.0&branch=pr-en-us-4170) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-5.1&branch=pr-en-us-4170) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-6&branch=pr-en-us-4170) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C17%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904172313122754-4170%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ZJbSQPdhHa10iOg8b3O4cVLGj4SBrWqzKnK8LCU4G8U%3D&st=2019-04-17T23%3A13%3A04Z&se=2019-05-18T23%3A18%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[fe2b796](https://github.com/sdwheeler/PowerShell-Docs/pull/4170/commits/fe2b7964c8b05610724860eedf08c572951e9659)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/toc.yml](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4146/reference/docs-conceptual/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-3.0&branch=pr-en-us-4170) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-4.0&branch=pr-en-us-4170) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-5.0&branch=pr-en-us-4170) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-5.1&branch=pr-en-us-4170) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-6&branch=pr-en-us-4170) (powershell-6) |
[reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-62.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4146/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-62.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-3.0&branch=pr-en-us-4170) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-4.0&branch=pr-en-us-4170) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-5.0&branch=pr-en-us-4170) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-5.1&branch=pr-en-us-4170) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-6&branch=pr-en-us-4170) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C17%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904172333303812-4170%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=gpplRi1ihGtGRfSshF6jser%2FUoTza0k%2FcmRFw4c1YTM%3D&st=2019-04-17T23%3A33%3A23Z&se=2019-05-18T23%3A38%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[601508a](https://github.com/sdwheeler/PowerShell-Docs/pull/4170/commits/601508a3d688f56d5ddee7ea70884374c4a9606c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/toc.yml](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4146/reference/docs-conceptual/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-3.0&branch=pr-en-us-4170) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-4.0&branch=pr-en-us-4170) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-5.0&branch=pr-en-us-4170) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-5.1&branch=pr-en-us-4170) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-6&branch=pr-en-us-4170) (powershell-6) |
[reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-62.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4146/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-62.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-3.0&branch=pr-en-us-4170) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-4.0&branch=pr-en-us-4170) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-5.0&branch=pr-en-us-4170) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-5.1&branch=pr-en-us-4170) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-6&branch=pr-en-us-4170) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C17%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904172338258335-4170%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=uhmU0rfxYWZfQlIZirNX0ghj6YdjHxLZs9Fdz7I06X0%3D&st=2019-04-17T23%3A38%3A16Z&se=2019-05-18T23%3A43%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[6164dcf](https://github.com/sdwheeler/PowerShell-Docs/pull/4170/commits/6164dcfd646d6e4d24c5ee6f5de9ec93ac1ef300)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/toc.yml](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4146/reference/docs-conceptual/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-3.0&branch=pr-en-us-4170) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-4.0&branch=pr-en-us-4170) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-5.0&branch=pr-en-us-4170) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-5.1&branch=pr-en-us-4170) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-6&branch=pr-en-us-4170) (powershell-6) |
[reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4146/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-3.0&branch=pr-en-us-4170) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-4.0&branch=pr-en-us-4170) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-5.0&branch=pr-en-us-4170) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-5.1&branch=pr-en-us-4170) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-6&branch=pr-en-us-4170) (powershell-6) |
[reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-62.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4146/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-62.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-3.0&branch=pr-en-us-4170) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-4.0&branch=pr-en-us-4170) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-5.0&branch=pr-en-us-4170) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-5.1&branch=pr-en-us-4170) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-6&branch=pr-en-us-4170) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C18%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904181445088404-4170%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=fPg28v%2BuGgVImjkgrFNlMEL%2FF3dNJjZaAZAtrmR6I2U%3D&st=2019-04-18T14%3A45%3A14Z&se=2019-05-19T14%3A50%3A14Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-62.md [hunk 1]
```diff
@@ -6,43 +6,136 @@ ms.date: 03/28/2019
...
+
+```Output
+PS> Import-AzRecoveryServicesAsrVaultSettingsFile
+```
+
+> [!NOTE]
+> This feature is intended to be used interactively. Abbreviated forms of cmdlets can't be executed.
+> This feature is not a replacement for aliases.
 
 ## Breaking Changes
```
Comment (joeyaiello) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-62.md]: There should be more of these if you continue scrolling through the changelog for previews. E.g.
> Make Join-String -InputObject 1,2,3 result equal to 1,2,3 | Join-String result (#8611) (Thanks @sethvs!)

is in 6.2.0-rc. And there's 4 in preview.4. 2 in preview.3, 4 in preview.2, and 1 in preview.1.

Did you happen to look at those for the rest of the content?

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[bac8cb9](https://github.com/sdwheeler/PowerShell-Docs/pull/4170/commits/bac8cb979daab31c946abfaf8f372a4c631d8498)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/toc.yml](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4146/reference/docs-conceptual/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-3.0&branch=pr-en-us-4170) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-4.0&branch=pr-en-us-4170) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-5.0&branch=pr-en-us-4170) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-5.1&branch=pr-en-us-4170) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-6&branch=pr-en-us-4170) (powershell-6) |
[reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4146/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-3.0&branch=pr-en-us-4170) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-4.0&branch=pr-en-us-4170) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-5.0&branch=pr-en-us-4170) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-5.1&branch=pr-en-us-4170) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-6&branch=pr-en-us-4170) (powershell-6) |
[reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-62.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4146/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-62.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-3.0&branch=pr-en-us-4170) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-4.0&branch=pr-en-us-4170) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-5.0&branch=pr-en-us-4170) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-5.1&branch=pr-en-us-4170) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-6&branch=pr-en-us-4170) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C18%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904182229410362-4170%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2FobldVBbJ4LWm%2FabcgUjUDz9RB9CizvPfBP4MleySno%3D&st=2019-04-18T22%3A29%3A41Z&se=2019-05-19T22%3A34%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[a63761e](https://github.com/sdwheeler/PowerShell-Docs/pull/4170/commits/a63761e76d7b2c3e564f131585ba883497f0474e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/toc.yml](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4146/reference/docs-conceptual/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-3.0&branch=pr-en-us-4170) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-4.0&branch=pr-en-us-4170) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-5.0&branch=pr-en-us-4170) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-5.1&branch=pr-en-us-4170) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-6&branch=pr-en-us-4170) (powershell-6) |
[reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4146/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-3.0&branch=pr-en-us-4170) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-4.0&branch=pr-en-us-4170) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-5.0&branch=pr-en-us-4170) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-5.1&branch=pr-en-us-4170) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-6&branch=pr-en-us-4170) (powershell-6) |
[reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-62.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4146/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-62.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-3.0&branch=pr-en-us-4170) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-4.0&branch=pr-en-us-4170) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-5.0&branch=pr-en-us-4170) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-5.1&branch=pr-en-us-4170) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-6&branch=pr-en-us-4170) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C19%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904191906045479-4170%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=GY%2FnKOk0VJ%2BPIbp6OR6TeFO7Pzi9DoP%2FYCT%2BCUigzZw%3D&st=2019-04-19T19%3A06%3A30Z&se=2019-05-20T19%3A11%3A30Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[a50f4a6](https://github.com/sdwheeler/PowerShell-Docs/pull/4170/commits/a50f4a6090fb240f6dc8727ffa7603a7ff23394b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/toc.yml](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4146/reference/docs-conceptual/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-3.0&branch=pr-en-us-4170) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-4.0&branch=pr-en-us-4170) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-5.0&branch=pr-en-us-4170) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-5.1&branch=pr-en-us-4170) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/web-access/troubleshooting-access-problems-in-windows-powershell-web-access?view=powershell-6&branch=pr-en-us-4170) (powershell-6) |
[reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4146/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-3.0&branch=pr-en-us-4170) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-4.0&branch=pr-en-us-4170) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-5.0&branch=pr-en-us-4170) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-5.1&branch=pr-en-us-4170) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-61?view=powershell-6&branch=pr-en-us-4170) (powershell-6) |
[reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-62.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4146/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-62.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-3.0&branch=pr-en-us-4170) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-4.0&branch=pr-en-us-4170) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-5.0&branch=pr-en-us-4170) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-5.1&branch=pr-en-us-4170) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/what-s-new-in-powershell-core-62?view=powershell-6&branch=pr-en-us-4170) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C19%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904192114447981-4170%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ynlDHzQ8y5v5rJCPMx0wxClID4cCFjHUd804TBtqsrk%3D&st=2019-04-19T21%3A14%3A42Z&se=2019-05-20T21%3A19%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**